src/algebra/associated.lean

```diff
@@ -748,3 +748,11 @@ dvd_and_not_dvd_iff.symm
 end comm_cancel_monoid_with_zero
 
 end associates
+
+namespace multiset
+
+lemma multiset.prod_ne_zero_of_prime [comm_cancel_monoid_with_zero α ][nontrivial α]
```

Comment:
The probable reason dot notation didn't work: because we're already in the `multiset` namespace, it would be called `multiset.multiset.prod_ne_zero_of_prime` instead:
```suggestion
lemma prod_ne_zero_of_prime [comm_cancel_monoid_with_zero α] [nontrivial α]
```